### PR TITLE
POL-370 Filter standardization

### PR DESCRIPTION
**Pull request by aferd**

### CODE REVIEW

File: src/components/Policy/TableToolbar/PolicyTableToolbar.tsx [hunk 1]
```diff
@@ -62,21 +61,19 @@ const filterMetadata: ColumnsMetada<typeof PolicyFilterColumn> = {
...
         placeholder: 'Filter by enabled status',
         options: {
-            default: 'all',
-            exclude: [ 'all' ],
-            exclusive: true,
+            exclusive: false,
             items: [
                 {
                     value: 'all',
                     label: <>All</>
```
Comment (josejulio) [inline src/components/Policy/TableToolbar/PolicyTableToolbar.tsx:68]: Is this one needed?

**Comment by josejulio** [comment]
Could you add a screenshot or gif of the result?

File: src/components/Policy/TableToolbar/PolicyTableToolbar.tsx (hunk 1 above)
Comment (aferd) [reply src/components/Policy/TableToolbar/PolicyTableToolbar.tsx:68]: Nope! I read part of the issue wrong, removing now!

**Comment by aferd** [comment]
After 
Screen Shot 2021-07-19 at 3 08 23 PM

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**